ui/jest.config.js

```diff
@@ -0,0 +1,13 @@
+/** @type {import('ts-jest/dist/types').InitialOptionsTsJest} */
+/* eslint-env node */
+module.exports = {
+  preset: 'ts-jest',
+  testEnvironment: 'jsdom',
+  transform: {
+    '^.+\\.(ts|tsx)?$': 'ts-jest',
+    "^.+\\.(js|jsx)$": "babel-jest",
```

Comment:
Why is Babel required?

Reply:
Yeah, so my understanding here is kind of shaky, but I believe in a normally TS only repo, you can get away with only using `ts-jest` to transform your typescript files into JS to send to `jest`. In `lila`, because of the way our modules work, `ts-jest` was erroring out when it encountered an already transpiled JS file, e.g. `ui/chess/dist/main.js`. In that case, apparently you need this other transformer `babel-jest`, which unfortunately relies on all the other babel stuff.